sdks/python/apache_beam/examples/inference/vertex_ai_image_classification.py

```diff
@@ -74,11 +74,24 @@ def parse_known_args(argv):
       type=str,
       required=True,
       help='GCP location for the Endpoint')
+  parser.add_argument(
+      '--endpoint_network',
+      dest='vpc_network',
+      type=str,
+      required=False,
+      help='GCP network the endpoint is peered to')
   parser.add_argument(
       '--experiment',
       dest='experiment',
+      type=str,
       required=False,
       help='GCP experiment to pass to init')
```

Comment:
Nit (missed previously): could we call out what experiment does here

Reply:
I believe the experiment is passing a string to enable this feature - https://cloud.google.com/vertex-ai/docs/experiments/intro-vertex-ai-experiments

I can add the context link here, or I can drop it into the model handler itself. Or both, depending on preference. It's not wholly necessary for the example though

Reply:
I'd say both is most helpful. If we're offering the option, we should describe it

Reply:
Wound up doing a little bit of both, better help message + a context link in the model handler docstring